wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/elevatorprofiledpid/Robot.java

```diff
@@ -40,6 +43,6 @@ public void teleopPeriodic() {
     }
 
     // Run controller and update motor output
-    m_motor.set(m_controller.calculate(m_encoder.getDistance()));
+    m_motor.setVoltage(m_controller.calculate(m_encoder.getDistance()) + m_feedforward.calculate(m_controller.getSetpoint().velocity));
```

Comment:
If output units are changed from duty cycle to volts, other values and docs need to be unit corrected.

Reply:
What docs?